lib/fluent/plugin/out_mongo.rb

```diff
@@ -172,7 +172,6 @@ def client
     def collect_records(chunk)
       records = []
       chunk.msgpack_each {|time, record|
-        record[@time_key] = Time.at(time || record[@time_key]) if @include_time_key
```

Comment:
How about this? https://github.com/fluent/fluent-plugin-mongo/pull/81/commits/638cf30bd0a3c6dc57ed74f0a177522843694623